website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +62,28 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
 
-<Doc::Layout @spacing="12px" @direction="vertical">
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px will take up too much of the screen space.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts should appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. Because of the placement of Toasts, any page-specific feedback to the user should be directly associated with the UI element. 
 
 ![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
 
 ### Displaying multiple Toasts
 
-When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast.
+When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast. For consistency, each stacked Toast should be the same width.
+
+![Multiple toast examples stacked](/assets/components/toast/toast-placement-multiple.png =600x*)
+
+## When to persist or time-out
+
+A Toast can either persist (until dismissed) or automatically dismiss (time-out) after seven seconds. If a Toast contains actions or critical information (often associated with warning or critical toasts, but not limited to), it is recommended to persist until the user dismisses it, otherwise, it should time-out after 7 seconds.
+
+### Time-out best practices
 
-![Multiple toast examples stacked](/assets/components/toast/toast-placement-multiple.png =600x*)
\ No newline at end of file
+When a Toast is set to time-out, the content within should be no longer than 5 words (e.g. “Cluster created”). It is recommended that the neutral, highlight or success colors be used for time-out experiences, ensuring that the criticality is at a minimum, and that users are not missing important information that requires their immediate attention.
```

Comment:
Where did "5 words" come from? Is this a strict rule? What if they have 6 words and can't omit 1 of the words? Should the number of words or characters change the length of time a Toast is visible?

Reply:
This was a conversation Lee and I had where she worked on a toast that would extend the timeout duration as more words were placed in the toast. She felt like that approach wasn't good, and personally I'm not keen on doing this because that gives folks a reason to add extended lines of text and a toast isn't a place for that.

But I get what you mean, ill massage this a bit to be a little more flexible in the wording and give consumers more grace.